src/Microsoft.DotNet.Build.Tasks/ZipFileCreateFromDependencyLists.cs

```diff
@@ -70,8 +70,8 @@ public override bool Execute()
             catch (Exception e)
             {
                 // We have 2 log calls because we want a nice error message but we also want to capture the callstack in the log.
-                Log.LogError($"An exception has occured while trying to compress files listed in {nameof(DependencyListFiles)} into '{DestinationArchive}'.");
-                Log.LogMessage(MessageImportance.Low, e.ToString());
+                Log.LogError($"An exception has occurred while trying to compress files listed in {nameof(DependencyListFiles)} into '{DestinationArchive}'.");
+                Log.LogErrorFromException(e, /*show stack=*/ true, /*show detail=*/ true, DestinationArchive);
```

Comment:
nit: instead of the comments I would just used name parameters so `showStackTrace:true, showDetail:true`